src/utils/DateUtils.js

```diff
@@ -131,22 +128,22 @@ module.exports = {
      * @param {Date}
      * @return {String}
      */
-    formatDate(date, format) {
+    formatDate(date, format, traslation) {
         let year = date.getFullYear()
         let month = date.getMonth() + 1
         let day = date.getDate()
-        let monthName = this.getMonthName(date.getMonth())
+        let monthName = this.getMonthName(date.getMonth(), traslation.months.original)
         let str = format
+            .replace(/S/, this.getNthSuffix(date.getDate()))
```

Comment:
I will Analyze how the suffixes will vary in different languages and how it could be adapted. Greetings.